### PR TITLE
DOC-2059: Added TINY-9793 to the release notes.

### PR DESCRIPTION
**Pull request by kemister85**
Ticket: DOC-2059

Changes:
* Added TINY-9793 to the release notes.

Pre-checks:
- [x] Branch prefixed with `feature/6/` or `hotfix/6/`
- [x] Changelog entry added
- [x] ~`modules/ROOT/nav.adoc` has been updated (if applicable)~
- [x] ~Files has been included where required (if applicable)~
- [x] ~Files removed have been deleted, not just excluded from the build (if applicable)~
- [x] ~(New product features only) Release Note added~

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: modules/ROOT/pages/6.5-release-notes.adoc [hunk 1]
```diff
@@ -73,6 +73,12 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 
 {productname} 6.5 also includes the following improvements:
 
+=== Screen readers now announce instructions for resizing the editor using arrow keys, when the resize handle is focused.
+//#TINY-9793
+
+In previous versions of {productname}, when using voice assistant software when focused on the editor's resize handle it would not announce any instructions to the user.
+
+In {productname} 6.5, the voice assistant software will now announce instructions  to the user for adjusting the size of the editor, by using directional arrow keys such as `↑` arrow up, `↓` arrow down ,  `→` arrow right,  and  `←` arrow left, when the editor resize handle is focused.
```
Comment (kemister85) [inline modules/ROOT/pages/6.5-release-notes.adoc:81]:
```suggestion
When using voice assistant software in previous versions of {productname}, focusing on the editor resize handle would not announce any instruction.

In version 6.5 instructions for adjusting the size of the editor using arrow keys are announced, when the editor resize handle is focused.
```

Comment (kemister85) [reply modules/ROOT/pages/6.5-release-notes.adoc:81]: @MitchCTiny reverted copy edits to reflect your entry in the TINY-9793 documentation section as requested.

File: modules/ROOT/pages/6.5-release-notes.adoc [hunk 2]
```diff
@@ -73,6 +73,12 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 
 {productname} 6.5 also includes the following improvements:
 
+=== Screen readers now announce instructions for resizing the editor using arrow keys, when the resize handle is focused.
+//#TINY-9793
+
+When using voice assistant software in previous versions of {productname}, focusing on the editor resize handle would not announce any instruction.
+
+In version 6.5 instructions for adjusting the size of the editor using arrow keys are announced, when the editor resize handle is focused.
```
Comment (MitchCTiny) [inline modules/ROOT/pages/6.5-release-notes.adoc:81]:
```suggestion
In {productname} 6.5, instructions for adjusting the size of the editor using arrow keys are now announced upon focusing the editor resize handle, while using voice assistant software.
```

**Review by MitchCTiny — APPROVED** [review]

File: modules/ROOT/pages/6.5-release-notes.adoc [hunk 3]
```diff
@@ -73,6 +73,12 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 
 {productname} 6.5 also includes the following improvements:
 
+=== Screen readers now announce instructions for resizing the editor using arrow keys, when the resize handle is focused.
+//#TINY-9793
+
+When using voice assistant software in previous versions of {productname}, focusing on the editor resize handle would not announce any instruction.
+
+In {productname} 6.5, instructions for adjusting the size of the editor using arrow keys are now announced upon focusing the editor resize handle, while using voice assistant software.
```
Comment (MitchCTiny) [inline modules/ROOT/pages/6.5-release-notes.adoc:81]:
```suggestion
In {productname} 6.5, instructions for adjusting the size of the editor using arrow keys are now announced upon focusing the editor resize handle, while using voice assistant software.

NOTE: In {productname} 6.5 Enterprise, this announcement is only available in English. Additional translations will be added in a future release.
```
There should be a known issue about the translations, both here and under "Known issues". Forgot about this until today, sorry.

**Review by MitchCTiny — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/6.5-release-notes.adoc [hunk 4]
```diff
@@ -119,4 +127,14 @@ In {productname} 6.5, the markup was fixed by replacing the div element with a `
...
+
+=== Screen readers are currently only supported in English ('en') when announcing instructions for resizing the editor.
+//#TINY-9793
+
+
+In {productname} 6.5, introduced announcing instructions for adjusting the editor's size using arrow keys to support screen readers.
+
+However, there is a known issue where this new instruction is currently only available in English ('en') when resizing the editor. 
+
+NOTE: that future releases of {productname} will include additional translations to provide broader support.
```
Comment (MitchCTiny) [inline modules/ROOT/pages/6.5-release-notes.adoc:140]:
```suggestion
=== Untranslated instructions for editor resize handle when using screen readers

In {productname} 6.5, instructions for using the editor resize handle were introduced, however these instructions are only provided in English.

Translations for these instructions will be added in a future enterprise release.
```
Can be trimmed down.

**Review by MitchCTiny — APPROVED** [review]

File: modules/ROOT/pages/6.5-release-notes.adoc [hunk 5]
```diff
@@ -73,6 +73,14 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
...
 {productname} 6.5 also includes the following improvements:
 
+=== Screen readers now announce instructions for resizing the editor using arrow keys, when the resize handle is focused.
+//#TINY-9793
+
+When using voice assistant software in previous versions of {productname}, focusing on the editor resize handle would not announce any instruction.
+
+In {productname} 6.5, instructions for adjusting the size of the editor using arrow keys are now announced upon focusing the editor resize handle, while using voice assistant software.
+
+NOTE: In {productname} 6.5 Enterprise, this announcement is only available in English. Additional translations will be added in a future release.
```
Comment (brian-forte) [inline modules/ROOT/pages/6.5-release-notes.adoc:83]:
```suggestion
=== Screen readers now announce instructions for using arrow keys to resize the editor when the resize handle is focused
//#TINY-9793

When using voice assistant software in previous versions of {productname}, focusing on the editor resize handle would not announce any instruction.

In {productname} 6.5, instructions for using arrow keys to adjust the editor viewport’s size are now announced upon focusing the editor resize handle, while using voice assistant software.

NOTE: In {productname} 6.5.1, this announcement is only available in English. Additional translations will be added in a future release.
```
Copy-edit for parse-ability.

**Review by brian-forte — APPROVED** [review]

**Pull request merged**